src/backend/distributed/master/master_repair_shards.c

```diff
@@ -237,6 +243,17 @@ RepairShardPlacement(int64 shardId, char *sourceNodeName, int32 sourceNodePort,
 								  "not supported.", relationName)));
 	}
 
+	/*
+	 * Let's not allow repairing partitions to prevent any edge cases.
+	 * We're already not allowing any kind of modifications on the partitions
+	 * so their placements are not likely to to be marked as INVALID. The only
+	 * possible case to mark placement of a partition as invalid is
+	 * "ALTER TABLE parent_table DETACH PARTITION partition_table". But,
```

Comment:
My first understanding of `regular` was `non-distributed` table here. Could you reword this such that it will mean `non-partitioned but still distributed` ?